### ISSUE TITLE
Metric lines is an internal metric computed by SonarQube. Provided value is ignored.

### ISSUE THREAD

**Issue by guwirth** [SonarOpenCommunity/sonar-cxx#828]
LOG file item:
```
DEBUG: Metric lines is an internal metric computed by SonarQube. Provided value is ignored.
```
Remove lines metric calculation from plugin it's part of core now.

**Comment by guwirth** [comment]
@all does someone know how to fix this?

**Comment by guwirth** [comment]
Solved with #913
